helpers/hcache_simple.py

```diff
@@ -461,7 +462,9 @@ def reset_mem_cache(func_name: str = "") -> None:
 
 
 def reset_disk_cache(func_name: str = "") -> None:
-    assert 0
+    assert hsystem.query_yes_no(
```

Comment:
Each line should do one thing, so you should run the system query and then assert.

Also I believe query_yes_no has an option to do this automatically if the answer is no.